packages/deployment/scripts/1-deploy-protocol/output/protocol.rinkeby.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
We don't have to do this now (let's leave it as an issue for when we do mainnet), but what do you think about re-deploying the "official" Rinkeby and Ropsten instances such that all the token and module addresses are the same?